src/main/java/io/syndesis/qe/marketplace/manifests/Opm.java

```diff
@@ -0,0 +1,81 @@
+package io.syndesis.qe.marketplace.manifests;
+
+import io.syndesis.qe.marketplace.quay.QuayUser;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.io.IOUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.Charset;
+
+@Slf4j
+public class Opm {
+
+    private File binary;
+    private static final String OPM_IMAGE = "registry.redhat.io/openshift4/ose-operator-registry:";
+    private static final String version = "v4.5";
```

Comment:
don't you want to get the same version as the openshift seerver you are currently using?